tests/shopify_python/test_google_styleguide.py

```diff
@@ -198,3 +198,30 @@ def fnc():
         message = pylint.testutils.Message('complex-list-comp', node=bad_list_comp)
         with self.assertAddsMessages(message):
             self.walk(root)
+
+    def test_use_cond_exprs(self):
+        # this parsed expression should send a message
+        root_to_msg = astroid.builder.parse("""
+            if 1 < 2:
+                num = 1
+            else:
+                num = 2
+            """)
+
+        # this parsed expression should NOT send a message
+        root_not_msg = astroid.builder.parse("""
+            if 1 < 2:
+                num = 1
+            else:
+                dum = 2
+            """)
```

Comment:
> wouldn't there need to be another value for other?

Yeah, oops.

Ah, `node.body[0].targets[0]` becomes a `Tuple`, so you get off easy.